pkg/descheduler/pod/pods.go

```diff
@@ -53,6 +54,28 @@ func ListPodsOnANode(ctx context.Context, client clientset.Interface, node *v1.N
 	return pods, nil
 }
 
+// GetPodOwnerReplicationCount returns the pod owner's replica count
+func GetPodOwnerReplicationCount(ctx context.Context, client clientset.Interface, ownerRef metav1.OwnerReference) (int, error) {
+	switch ownerRef.Kind {
+	case "ReplicaSet":
+		owner, err := client.AppsV1().ReplicaSets(v1.NamespaceAll).Get(ctx, ownerRef.Name, metav1.GetOptions{})
```

Comment:
Maybe better to use pod's namespace here?

Reply:
> Maybe better to use pod's namespace here?

Yeah ... I think this makes sense to me. Use the pods namespace instead of `v1.NamespacesAll`. @hanumanthan can you make this change?